src/test/resources/features/delete_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Accounts
+
+  Background:
+    Given I sign in as "owner"
+    And I click my username menu
+    And I click "accounts" menu link
+    And I click the create account button
+    And I enter the account name "AccountTest"
+    And I click create button
+
+  Scenario: Delete Account
+    When I click the "Settings" tab button
+    And I click the "delete this account" link
+    And I click the OK button to confirm
+    Then I validate the confirmation label is "AccountTest was successfully deleted."
```

Comment:
It is missing validation in the account list, account dropdown in create project and account dropdown in project setting form.

Reply:
Oh I didn't realize this. I will do it right away...